model/schema/allele.yaml

```diff
@@ -464,6 +560,14 @@ classes:
       - agm_curie
       - evidence_curies
 
+
+  AlleleGenomicLocationAssociation:
+    is_a: GenomicLocationAssociation
```

Comment:
`GenomicLocationAssociation` doesn't exist.  I think this should just be '`is_a: Association`.

Reply:
@chris-grove Wouldn't it remain as a `GenomicLocation` object with classes such as `AlleleGenomicLocation` linking it to an allele (or whatever it relates to)

Reply:
@markquintontulloch 
The current GenomicLocation class is modeled like an association with 'subject', 'predicate', and 'object' slots and only lacks 'evidence', which we probably want anyway, making me think that this should just be an Association subclass where the slots 'subject', 'predicate', and 'object' do not need to be declared.

Reply:
@chris-grove I think I follow you.  So `GenomicLocationAssociation` would have the various properties such as assembly, etc. and AlleleGenomicLocationAssociation would just specify that the subject is an Allele? 

I was thinking that there would be a GenomicLocation object that had a `chromosome` slot rather than an association with Chromosome as the `object`, but you're right - that's not how it's currently modelled so best to stick with that paradigm.  I think there's some redundancy in the modelling here as presumably a Chromosome object would have an assembly, so no need for the `has_assembly` slot on `GenomicLocation`, but I don't think we've really looked at this area yet so there's probably lots of changes/updates in the pipeline.

Reply:
@markquintontulloch 
Looking at the Neo4J model, assembly is a direct property of the GenomicLocation class and I imagine is based on this agr_schemas model:
https://github.com/alliance-genome/agr_schemas/blob/master/ingest/allele/variantDetail.json
I would be inclined to stick with associating assembly with the genomic location association object unless we are clear on the original logic of why it was structured that way and we can verify we no longer rely on it to be that way.

Reply:
I certainly don't think we should hold up this PR on the basis of sorting out GenomicLocation.